packages/quick_actions/quick_actions_ios/test/quick_actions_ios_test.dart

```diff
@@ -2,171 +2,89 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-import 'dart:async';
-
-import 'package:flutter/services.dart';
 import 'package:flutter_test/flutter_test.dart';
+import 'package:quick_actions_ios/messages.g.dart';
 import 'package:quick_actions_ios/quick_actions_ios.dart';
 import 'package:quick_actions_platform_interface/quick_actions_platform_interface.dart';
 
+const String LAUNCH_ACTION_STRING = 'aString';
+
 void main() {
   TestWidgetsFlutterBinding.ensureInitialized();
 
-  group('$QuickActionsIos', () {
-    late List<MethodCall> log;
+  final _FakeQuickActionsApi api = _FakeQuickActionsApi();
+  final QuickActionsIos quickActions = QuickActionsIos(api: api);
 
-    setUp(() {
-      log = <MethodCall>[];
-    });
-
-    QuickActionsIos buildQuickActionsPlugin() {
-      final QuickActionsIos quickActions = QuickActionsIos();
-      _ambiguate(TestDefaultBinaryMessengerBinding.instance)!
-          .defaultBinaryMessenger
-          .setMockMethodCallHandler(quickActions.channel,
-              (MethodCall methodCall) async {
-        log.add(methodCall);
-        return '';
-      });
-
-      return quickActions;
-    }
+  test('registerWith() registers correct instance', () {
+    QuickActionsIos.registerWith();
+    expect(QuickActionsPlatform.instance, isA<QuickActionsIos>());
+  });
 
-    test('registerWith() registers correct instance', () {
-      QuickActionsIos.registerWith();
-      expect(QuickActionsPlatform.instance, isA<QuickActionsIos>());
+  group('#initialize', () {
+    test('initialize', () async {
+      await quickActions.initialize((_) {});
     });
+  });
 
-    group('#initialize', () {
-      test('passes getLaunchAction on launch method', () {
-        final QuickActionsIos quickActions = buildQuickActionsPlugin();
-        quickActions.initialize((String type) {});
-
-        expect(
-          log,
-          <Matcher>[
-            isMethodCall('getLaunchAction', arguments: null),
-          ],
-        );
-      });
-
-      test('initialize', () async {
-        final QuickActionsIos quickActions = buildQuickActionsPlugin();
-        final Completer<bool> quickActionsHandler = Completer<bool>();
-        await quickActions
-            .initialize((_) => quickActionsHandler.complete(true));
-        expect(
-          log,
-          <Matcher>[
-            isMethodCall('getLaunchAction', arguments: null),
-          ],
-        );
-        log.clear();
-
-        expect(quickActionsHandler.future, completion(isTrue));
-      });
-    });
+  test('setShortCutItems', () async {
```

Comment:
Nit: we can fix the capitalization here, since at this layer it's correctly `Shortcut`.